data/presets/amenity/parcel_locker.json

```diff
@@ -1,4 +1,7 @@
 {
+    "aliases": [
+        "package locker"
```

Comment:
Unlike terms, aliases are Title Cased just like the preset name.

```suggestion
        "Package Locker"
```

Reply:
Ah thanks for the heads up, should be fixed now.